compiler/rustc_errors/src/emitter.rs

```diff
@@ -2824,6 +2829,19 @@ impl Style {
     }
 }
 
+/// Whether the original and suggested code are the same.
+pub fn is_no_change(sm: &SourceMap, suggested: &str, sp: Span) -> bool {
+    // FIXME: this should probably be extended to also account for `FO0` → `FOO` and unicode.
```

Comment:
nit: not sure I quite follow this comment: `FO0` and `FOO` are different and should be treated differently but something like whitespace only changes is probably not important for this use case.

Reply:
I *think* what I meant here is to take those differences and instead make the span smaller, but the comment as is doesn't make that argument. I'll remove it.